compute_endpoint/globus_compute_endpoint/endpoint/config/utils.py

```diff
@@ -7,6 +7,7 @@
 import pathlib
 import shlex
 
+import jsonschema
 import yaml
```

Comment:
Similar to [PR #1167](https://github.com/funcx-faas/funcX/pull/1167#discussion_r1221756909), this is only utilized by the EP Manager, right?  Suggest importing it inside the method that utilizes it.